pkgs/development/python-modules/screed/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, fetchPypi
+, buildPythonPackage
+, pythonOlder
+, bz2file
+, setuptools
+, setuptools_scm
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "screed";
+  version = "1.1";
+  disabled = pythonOlder "3.8";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-TDxF6AePMFY7areAJ9/++HSrEgdiWj7rVf8TooJthCY=";
+  };
+
+  buildInputs = [ setuptools_scm ];
+  SETUPTOOLS_SCM_PRETEND_VERSION = version;
+
+  pythonImportsCheck = [ "screed" ];
+  checkInputs = [ pytestCheckHook ];
+  disabledTests = [
+    "Test_convert_shell"
+    "Test_fa_shell_command"
+    "Test_fq_shell_command"
```

Comment:
Please add a short comment why those are disabled

Reply:
Done. I'll fix upstream for the next release and make sure they work in nix too.